### PR TITLE
simplified configuration for certificate choice, managed certs is the new default

### PR DESCRIPTION
**Pull request by danistrebel**
What's changed, or what was fixed?

- managed certificates are the default choice
- Hostnames are automatically set on the env groups

**Fixes:** #issue

- [x] I have run all the tests locally and they all pass.
- [x] I have followed the relevant style guide for my changes.

**CC:** @apigee-devrel-reviewers

### CODE REVIEW

**Review by laughingbiscuit — APPROVED** [review]

File: tools/apigee-x-trial-provision/apigee-x-trial-provision.sh [hunk 1]
```diff
@@ -74,38 +73,52 @@ export ZONE
 export AX_REGION
 export SUBNET=${SUBNET:-default}
 export ENVOY_MACHINE_TYPE=${ENVOY_MACHINE_TYPE:-e2-micro}
-export ENVOY_PREEMPTIBLE=${ENVOY_PREEMPTIBLE:-true}
-export MANAGED_CERTS=${MANAGED_CERTS:-false}
+export ENVOY_PREEMPTIBLE=${ENVOY_PREEMPTIBLE:-false}
```
Comment (srinandan) [inline tools/apigee-x-trial-provision/apigee-x-trial-provision.sh]: should we use a different name? There is no `ENVOY` being used.

Comment (danistrebel) [reply tools/apigee-x-trial-provision/apigee-x-trial-provision.sh]: We already do. See the 2nd commit.

**Pull request merged**